components/mailer.rst

```diff
@@ -177,6 +177,43 @@ the application or when using a self-signed certificate::
 
     The ``verify_peer`` option was introduced in Symfony 5.1.
 
+Other Options
+-------------
+
+.. versionadded:: 5.2
+
+    These options were added in Symfony 5.2.
+
+command
+~~~~~~~
+
+Command to be executed by ``sendmail`` transport.
+
+    $dsn = 'sendmail://default?command=/usr/sbin/sendmail%20-oi%20-t'
+
+local_domain
+~~~~~~~~~~~~
+
+The domain name to use in ``HELO`` command.
+
+    $dsn = 'smtps://smtp.example.com?local_domain=example.org'
+
+restart_threshold
+~~~~~~~~~~~~~~~~~
+
+The maximum number of messages to send before re-starting the transport. Can be 
+used together with ``restart_threshold_sleep`` which is the number of seconds to 
+sleep between stopping and re-starting the transport.
+
+    $dsn = 'smtps://smtp.example.com?restart_threshold=10&restart_threshold_sleep=1'
+
+ping_threshold
+~~~~~~~~~~~~~~
+
+The minimum number of seconds between two messages required to ping the server.
```

Comment:
```suggestion
The minimum number of seconds between two messages required to ping the server::
```

For your information, the double colon `::` enables PHP syntax highlighting for the following code 👍

Reply:
> For your information, the double colon `::` enables PHP syntax highlighting for the following code 👍

Didn't know that, thank you. I am still unfamiliar with Symfony's docs system :)